qiskit/pulse/pulse_lib/discrete.py

```diff
@@ -164,7 +164,8 @@ def sin(duration: int, amp: complex, freq: float = None,
 _sampled_gaussian_pulse = samplers.midpoint(continuous.gaussian)
 
 
-def gaussian(duration: int, amp: complex, sigma: float, name: Optional[str] = None) -> SamplePulse:
+def gaussian(duration: int, amp: complex, sigma: float, name: Optional[str] = None,
+             zero_ends: Optional[bool] = True) -> SamplePulse:
```

Comment:
```suggestion
             zero_ends: bool = True) -> SamplePulse:
```
Same for others as well :)